test/gtest/ucp/ucp_test.h

```diff
@@ -98,10 +114,10 @@ class ucp_test : public ucp_test_base,
     virtual void init();
     virtual void cleanup();
     entity* create_entity(bool add_in_front = false);
-    void progress() const;
-    void short_progress_loop() const;
+    void progress(int index) const;
```

Comment:
the parameter name should be "int worker" or "worker_index" because it's the index of the worker
also we can add default value of this parameter (c++) to avoid changing all tests